sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/Generated/ConversationAuthoringClient.cs

```diff
@@ -1440,26 +1214,23 @@ public virtual Pageable<BinaryData> GetModelEvaluationResults(string projectName
         /// </list>
         /// </summary>
         /// <param name="projectName"> The name of the project to use. </param>
-        /// <param name="maxCount"> The maximum number of resources to return from the collection. </param>
-        /// <param name="skip"> An offset into the collection of the first resource to be returned. </param>
-        /// <param name="maxpagesize"> The maximum number of resources to include in a single response. </param>
```

Comment:
Is this a generator change or did they remove these from the swagger spec?

Reply:
Actually, looking at this again, I think it's just producing a weird diff with another operation.